src/components/course/data/utils.jsx

```diff
@@ -405,7 +401,8 @@ export const getSubscriptionDisabledEnrollmentReasonType = ({
   subscriptionLicense,
   hasEnterpriseAdminUsers,
 }) => {
-  const hasExpiredSubscriptionLicense = customerAgreement?.netDaysUntilExpiration <= 0;
+  if (!subscriptionLicense) { return undefined; }
+  const hasExpiredSubscriptionLicense = !subscriptionLicense.subscriptionPlan.isCurrent;
```

Comment:
[inform] This could also be related to the bug fix for https://2u-internal.atlassian.net/browse/ENT-8918 as well.